13/main/index.html

```diff
@@ -0,0 +1,1480 @@
+        <h4 class="infoHeader" style="background-image:url('images_bw/threads_thumb.jpg');border-radius:90px;">Threads</h4>
+                <div class="content cf">
+                <div class="column-three last">
+                <h1 class="center">Threads</h1>
+                </div>
+                        <div class="column">
+                <span id="threads_image"><img alt="threads" src="images/threads.jpg" /></span>
+                        </div>
+                        <div class="column-two last" >
+                                <h3 id="threads_header">Threads @ Felicity</h3>
+                                <p id="threads_tagline" class="tagline">For geeks and humans</p>
+                                <p class="bottomSection" id="threads_content">Threads at Felicity is a one stop place to exercise all your programming and puzzle solving skills to test. Sponsored by Microsoft, this is one of the largest events of it's kind in India with 66 participant countries. <br /> Schedule : <br />Optimust : 16th Feb 8:00pm <br />CodeCraft : 17th Feb 2:00pm <br /> Gordian Knot: 20th Feb 8:00pm <br /> Break-in 18th Feb 8:00pm </p>
+                        </div>
+                        <div class="column">
+                        <h4 class="center">Programing</h4>
+                        <p class="center">
+                                <span class="event_name" onclick="change_frame('threads','codecraft')"> Code Craft </span> <br />
+                                <span class="event_name" onclick="change_frame('threads','tle')"> TLE </span><br />
+                                <span class="event_name" onclick="change_frame('threads','aichallenge')"> AI Challenge</span><br />
+
+
+                        </p>
+                        </div>
+                        <div class="column">
+                        <h4 class="center">Puzzles</h4>
+                        <p class="center">
+
+                                <span class="event_name" onclick="change_frame('threads','cachein')"> Cache-In </span> <br />
+                                <span class="event_name" onclick="change_frame('threads','gordian')"> Gordian-knot </span><br />
+                                <span class="event_name" onclick="change_frame('threads','dbug')"> Dbug the Cbug </span><br />
+                        </p>
+                        </div>
+                        <div class="column last">
+                        <h4 class="center">Advanced</h4>
+                        <p class="center bottomSection">
+                                <span class="event_name" onclick="change_frame('threads','seo')"> SEO </span> <br />
+                                <span class="event_name" onclick="change_frame('threads','breakin')"> Break-in </span> <br />
+
+
+
+
+                        </p>
+                        </div>
+                </div>
+        </div>
+
+
+
+
+
+
+<!-- Pulsation -->
```

Comment:
@nisargjhaveri 
@polybuildr 
@hharchani 

I now know where you got your "One Page To Rule Them All" idea. :grin: